src/components/data_search/DatasetSearchTable.jsx

```diff
@@ -104,60 +104,48 @@ export const DatasetSearchTable = (props) => {
       queryChunks.push(...searchModifier);
     }
 
-    var filterQuery = {};
+    let filterQuery = {};
     if (numSelectedFilters(filters) > 0) {
-      const shouldTerms = [];
+      const filterTerms = [];
 
       filters.accessManagement.forEach(term => {
-        shouldTerms.push({
+        filterTerms.push({
           'term': {
             'accessManagement': term
           }
         });
       });
 
       filters.dataUse.forEach(term => {
-        shouldTerms.push({
+        filterTerms.push({
           'match': {
             'dataUse.primary.code': term
           }
         });
       });
 
-      if (shouldTerms.length > 0) {
+      if (filterTerms.length > 0) {
         filterQuery = [
           {
             'bool': {
-              'should': shouldTerms
+              'must': filterTerms
             }
           }
         ];
       }
     }
 
-    // do not add filter subquery if no filters are applied
-    if (numSelectedFilters(filters) > 0) {
-      return {
-        'from': 0,
-        'size': 10000,
-        'query': {
-          'bool': {
-            'must': queryChunks,
-            'filter': filterQuery
-          }
-        }
-      };
-    } else {
-      return {
-        'from': 0,
-        'size': 10000,
-        'query': {
-          'bool': {
-            'must': queryChunks
-          }
+    return {
```

Comment:
This change wasn't required, but I thought it would be nice  to
- combine the code fragments to avoid duplication
- test `filterQuery` instead of recomputing `numSelectedFilters(filters)`, since `filterQuery` is the value actually used in the code block